ui/pages/bridge/prepare/bridge-cta-button.tsx

```diff
@@ -163,7 +160,7 @@ export const BridgeCTAButton = ({
       return t('bridgeFetchNewQuotes');
     }
     return undefined;
-  }, [wasTxDeclined, isQuoteExpired]);
+  }, [wasTxDeclined, isQuoteExpired, t]);
```

Comment:
Odd that you need `t` here. But I suppose it doesn't hurt?

Reply:
`t` comes from a useContext hook, so is flagged for the dep array rule.
```
const t = useI18nContext();
```

If imported directly, then we would not need to provide this in the dep array.

Both are fine.